tests/countries/test_solomon_islands.py

```diff
@@ -0,0 +1,257 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+
+from unittest import TestCase
+
+from holidays.countries.solomon_islands import SolomonIslands, SB, SLB
+from tests.common import CommonCountryTests
+
+
+class TestSolomonIslands(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1979, 2050)
+        super().setUpClass(SolomonIslands, years=years, years_non_observed=years)
+        cls.subdiv_holidays = {
+            subdiv: SolomonIslands(subdiv=subdiv, years=years)
+            for subdiv in SolomonIslands.subdivisions
+        }
+        cls.subdiv_holidays_non_observed = {
+            subdiv: SolomonIslands(subdiv=subdiv, years=years, observed=False)
+            for subdiv in SolomonIslands.subdivisions
+        }
+
+    def test_country_aliases(self):
+        self.assertAliases(SolomonIslands, SB, SLB)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(SolomonIslands(years=1978))
+
+    def test_special_holidays(self):
+        name = "Public Holiday"
+        dt = ("2022-09-12", "2024-04-17")
```

Comment:
```suggestion
        dt = (
            "2022-09-12",
            "2024-04-17",
        )
```

Reply:
Works for me 🤷‍♂️
[c59379a](https://github.com/vacanza/holidays/pull/2632/commits/c59379a8c1daf1afc674f04483b142595eb3f6c0)